examples/examples.md

```diff
@@ -956,32 +955,32 @@ The data output of the workflow contains the information of the exception caught
           }
        }
     ],
-    "onError": [
+    "stateDataFilter": {
+       "dataOutputPath": "{{ $.exceptions }}"
+    },
+    "transition": {
+       "nextState":"ApplyOrder"
+    },
+    "onErrors": [
        {
-         "expression": "{{ $.exceptions[?(@.name == 'MissingOrderIdException')] }}",
+         "error": "Missing order id",
```

Comment:
the mapping of domain specific errors + optional error codes are up to runtimes to provide. 
given different runtimes for sw this cannot be defined within our specification.
this is a common thing that many of the process runtime engines already are doing, not something we are 
making up

Reply:
but the way we define actions, we don't interact with language-specific runtimes, but with an OpenAPI spec

Reply:
okay, maybe "Missing order id" is misleading here. This would be something I'd expect as a response from the OpenAPI-based action. If it's runtime-specific, then maybe "OutOfMemory" is a more fitting example?